cloud_auto.yaml

```diff
@@ -32,9 +37,14 @@ Mappings:
   AWSRegionArch2AMI:
     us-east-1:
       HVM64: ami-0ff8a91507f77f867
+  Region2ARNPrefix:
+    us-east-1:
+      ARNPrefix: 'arn:aws:'
+  Region2ELBAccountId:
+    us-east-1:
+      AccountId: '127311923021'
```

Comment:
It isn't necessary to hardcode your AccountId and arn prefix. lets use built in variables AWS::Region, AWS::AccountId. See more in https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html
Usage: !Ref "AWS::Region"